src/libYARP_dev/src/yarp/dev/IJointBrake.h

```diff
@@ -0,0 +1,78 @@
+/*
+ * SPDX-FileCopyrightText: 2025-2025 Istituto Italiano di Tecnologia (IIT)
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#ifndef YARP_DEV_IJOINTBRAKE_H
+#define YARP_DEV_IJOINTBRAKE_H
+
+#include <yarp/dev/api.h>
+
+namespace yarp::dev {
+class IJointBrakeRaw;
+class IJointBrake;
+}
+
+/**
+ * @ingroup dev_iface_motor
+ *
+ * Interface for controlling a joint equipped with brakes (hardware or simulated).
+ */
+class YARP_dev_API yarp::dev::IJointBrake
+{
+public:
+    virtual ~IJointBrake(){}
+
+    /**
+    * Check is the joint brake is currently active.
+    * @param j joint number
+    * @param braked is true if the joint is currently braked
+    * @return: true/false success failure.
+    */
+    virtual yarp::dev::ReturnValue isJointBraked(int j, bool& braked) const =0;
+
+    /**
+    * Enables/Disables manually the joint brake.
```

Comment:
Does it make sense to add more detail about the expected behavior of brakes? For instance, what happens if one is set while the other is active?